From 6cae90cb48e7a84d6938a59c7638d32a8b279a02 Mon Sep 17 00:00:00 2001
From: Greg Still <stillgs@us.ibm.com>
Date: Fri, 27 Oct 2017 06:06:35 -0500
Subject: [PATCH 4/4] PM: Disable core hang buster

- Make default attribute control be DISABLED

Change-Id: I77bab151700247f7ea1ab6e77529db2bb5681fcb
CQ: SW406324
Reviewed-on: http://ralgit01.raleigh.ibm.com/gerrit1/48925
Tested-by: Jenkins Server <pfd-jenkins+hostboot@us.ibm.com>
Tested-by: HWSV CI <hwsv-ci+hostboot@us.ibm.com>
Tested-by: PPE CI <ppe-ci+hostboot@us.ibm.com>
Tested-by: Hostboot CI <hostboot-ci+hostboot@us.ibm.com>
Tested-by: FSP CI Jenkins <fsp-CI-jenkins+hostboot@us.ibm.com>
Reviewed-by: Brian T. Vanderpool <vanderp@us.ibm.com>
Dev-Ready: Gregory S. Still <stillgs@us.ibm.com>
Reviewed-by: Gregory S. Still <stillgs@us.ibm.com>
Reviewed-by: Jennifer A. Stofer <stofer@us.ibm.com>
---
 .../chips/p9/procedures/xml/attribute_info/pm_plat_attributes.xml       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/import/chips/p9/procedures/xml/attribute_info/pm_plat_attributes.xml b/src/import/chips/p9/procedures/xml/attribute_info/pm_plat_attributes.xml
index a77e93f..4b425fb 100644
--- a/src/import/chips/p9/procedures/xml/attribute_info/pm_plat_attributes.xml
+++ b/src/import/chips/p9/procedures/xml/attribute_info/pm_plat_attributes.xml
@@ -978,7 +978,7 @@
     <valueType>uint8</valueType>
     <enum>OFF=0, ON=1</enum>
     <platInit/>
-    <initToZero/>
+    <default>ON</default>
     <overrideOnly/>
   </attribute>
   <!-- ********************************************************************* -->
-- 
2.9.2.windows.1

